tests/actions/settingEncounterRole.test.js

```diff
@@ -7,6 +7,7 @@ import {
     SETTING_ENCOUNTER_ROLE_SUCCESS,
     SETTING_ENCOUNTER_ROLE_FAILURE,
     SETTING_ENCOUNTER_ROLE_LOADING,
+    FETCH_ENCOUNTER_ROLE_LOADING,
```

Comment:
@larrystone, in my opinion, testing `FETCH_ENCOUNTER_ROLE` in this file makes this an integrated test instead of a unit test. You could think of mocking the fetchEncounterole action.

Reply:
Thanks for bringing this up. I have fixed this